crowbar_framework/spec/controllers/deploy_queue_controller_spec.rb

```diff
@@ -64,19 +64,19 @@
     describe "with non-existing nodes" do
       before do
         # Cluster referencing a non-existent node (deleted)
-        ServiceObject.stubs(:expand_nodes_for_all).returns([["I just dont exist"],[]])
+        allow(ServiceObject).to receive(:expand_nodes_for_all).and_return([["I just dont exist"],[]])
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceAfterComma: Space missing after comma. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)